.github/workflows/code-quality.yml

```diff
@@ -36,7 +36,9 @@ jobs:
 
       - name: Install Arm Build Dependencies
         if: matrix.arch == 'arm64'
-        run: sudo apt-get install -qyy -o APT::Install-Suggests=false libgdal-dev
+        run: |
+          sudo apt-get update -qy
+          sudo apt-get install -qyy -o APT::Install-Suggests=false libgdal-dev
```

Comment:
should the `-qyy` here be `-qqy`? (I can't find anything documented on the double `y`, though could have missed it)

Reply:
Huh I can't find anything about `-yy`, maybe it was a typo